internal/blockchain/utxobackend.go

```diff
@@ -829,7 +844,7 @@ func (l *levelDbUtxoBackend) PutUtxos(utxos map[wire.OutPoint]*UtxoEntry,
 }
 
 // Upgrade upgrades the UTXO backend by applying all possible upgrades
-// iteratively as needed.
+// iteractively as needed.
```

Comment:
`iteratively` was correct.  As in one at a time.

Reply:
Will revert.

Reply:
reverted